{
  "config": {
    "title": "Ingeniumhass",
    "step": {
      "user": {
        "title": "Connect to Ingeniumhass",
        "data": {
          "host": "Host"
        }
      }
    },
    "error": {
      "cannot_connect": "Failed to connect, please try again",
      "invalid_auth": "Invalid authentication",
      "unknown": "Unexpected error"
    },
    "abort": {
      "already_configured": "Device is already configured"
    }
  }
}